docs/doc_helpers.bzl

```diff
@@ -0,0 +1,84 @@
+load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+
+def stardoc_with_diff_test(
```

Comment:
incidentally we have this function in aspect-build/bazel-lib now, and that repo is Apache 2, so we could possibly introduce a dependency on it from rules_go if you want to avoid maintaining this

Reply:
I'd like to keep deps in `rules_go` to a bare minimum for end users so I think keeping it in repo is not a terrible situation. Have you considered getting it into the `stardoc` repo? Then using it would be straight forward and wouldn't add any more deps.

Reply:
I haven't tried, based on Googlers silent on my https://github.com/bazelbuild/bazel-skylib/pulls/alexeagle and especially https://github.com/bazelbuild/bazel-skylib/pull/321 I don't want to spend the time if it's never going to merge.

Thanks for getting this in!